File: app/src/main/java/com/example/simonastojanovic/memeapp/injection/ActivityBindings.java
package com.example.simonastojanovic.memeapp.injection;

import com.example.simonastojanovic.memeapp.view.MainActivity;
import com.example.simonastojanovic.memeapp.view.MainActivityModule;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;

@Module
public abstract class ActivityBindings {

    @ContributesAndroidInjector(modules = MainActivityModule.class)
    abstract MainActivity bindMainActivity();

}

File: app/src/main/java/com/example/simonastojanovic/memeapp/injection/ActivityBindings.kt
package com.example.simonastojanovic.memeapp.injection

import com.example.simonastojanovic.memeapp.view.MainActivity
import com.example.simonastojanovic.memeapp.view.MainActivityModule

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class ActivityBindings {

    @ContributesAndroidInjector(modules = arrayOf(MainActivityModule::class))
    abstract fun bindMainActivity(): MainActivity

}
